packages/components/src/components/sequence/CallLabel.vue

```diff
@@ -64,7 +69,14 @@ export default {
   },
 
   computed: {
+    classes(): string[] {
+      const result = ['label'];
+      if (this.interactive) result.push('interactive');
+      return result;
+    },
```

Comment:
This can also be
```ts
{
  classes(): { [key: string]: boolean } {
    return {
      label: true,
      interactive: this.interactive,
    };
  }
}
```
It'll use every key name where the value is truthy.